### PR TITLE
core(emulation): retire moto g4, use moto g power

### LINKED ISSUE #14256: Update mobile device preset to match newer devices
https://twitter.com/patrickhulce/status/1554163245937737730

viewport, DPR, UA (but not CPU multiplier)

see also: https://github.com/GoogleChrome/lighthouse/issues/10741#issuecomment-626903508 about DPR. tldr: while lighthouse made some bad calls if our reference device was dpr=3, we (conceivably) fixed that so that shouldn't be a problem anymore.

### PR DESCRIPTION
**Pull request by connorjclark**
ref #14256

### CODE REVIEW

**Comment by adamraine** [comment]
We're also going to need to update DT settings when this is rolled since DT sets up emulation on it's own

File: core/config/constants.js [hunk 1]
```diff
@@ -52,14 +52,13 @@ const throttling = {
...
  */
-const MOTOG4_EMULATION_METRICS = {
+const MOTOGPOWER_EMULATION_METRICS = {
   mobile: true,
-  width: 360,
-  height: 640,
-  // Moto G4 is really 3, but a higher value here works against
-  // our perf recommendations.
+  width: 412,
+  height: 767,
```
Comment (paulirish) [inline core/config/constants.js:58]: did this include when you scrolled and the omnibox was offscreen? (i'm guessing so cuz this is actually a comparatively skinny aspect ratio)

767 feels kinda awk to me.  but i looked at the phones in CDT's emulated devices.. their aspect ratios... heights.. And tbh i dont think i can make a strong argument to change the number.  So yah. sg.

image

**Review by adamraine — COMMENTED** [review]
Should we update the user agent as well?

File: core/config/constants.js (hunk 1 above)
Comment (connorjclark) [reply core/config/constants.js:58]: > did this include when you scrolled and the omnibox was offscreen? (i'm guessing so cuz this is actually a comparatively skinny aspect ratio)

No, the omnibox is on the screen.

image

File: core/config/constants.js [hunk 2]
```diff
@@ -76,12 +75,12 @@ const DESKTOP_EMULATION_METRICS = {
...
 
 const screenEmulationMetrics = {
-  mobile: MOTOG4_EMULATION_METRICS,
+  mobile: MOTOGPOWER_EMULATION_METRICS,
   desktop: DESKTOP_EMULATION_METRICS,
 };
 
 
-const MOTOG4_USERAGENT = 'Mozilla/5.0 (Linux; Android 7.0; Moto G (4)) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/109.0.0.0 Mobile Safari/537.36'; // eslint-disable-line max-len
+const MOTOG4_USERAGENT = 'Mozilla/5.0 (Linux; Android 11; moto g power (2022)) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/109.0.0.0 Mobile Safari/537.36'; // eslint-disable-line max-len
```
Comment (adamraine) [inline core/config/constants.js:83]: Should we capitalize `moto g power`?

File: core/config/constants.js (hunk 1 above)
Comment (paulirish) [reply core/config/constants.js:58]: You should test on a page with a scrollable content and scroll down a bit to get the omnibox offscreen. Im pretty sure the other device preset heights are consistent with this.

File: core/config/constants.js (hunk 2 above)
Comment (connorjclark) [reply core/config/constants.js:83]: this is how it appeared on my device

File: core/config/constants.js (hunk 1 above)
Comment (connorjclark) [reply core/config/constants.js:58]: ok, that gets me 412x823

File: core/config/constants.js (hunk 2 above)
Comment (adamraine) [reply core/config/constants.js:83]: works for me

Comment (benschwarz) [reply core/config/constants.js:83]: fyi we went with `Mozilla/5.0 (Linux; Android 11; moto g power) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/%s Mobile Safari/537.36`

File: cli/test/smokehouse/test-definitions/dobetterweb.js [hunk 3]
```diff
@@ -355,9 +355,9 @@ const expectations = {
           items: {
             0: {
               displayedAspectRatio: /^120 x 15/,
-              url: 'http://localhost:10200/dobetterweb/lighthouse-480x318.jpg?iar1',
+              url: 'http://localhost:10200/dobetterweb/lighthouse-1024x680.jpg?iar1',
             },
-            length: 1,
+            length: 2,
```
Comment (adamraine) [inline cli/test/smokehouse/test-definitions/dobetterweb.js:360]: What's the new one, should we add an expectation for it now?

File: cli/test/smokehouse/test-definitions/fps-max.js [hunk 4]
```diff
@@ -18,12 +18,12 @@ const config = {
...
-      innerHeight: 640,
-      outerWidth: 360,
-      outerHeight: 640,
+      innerWidth: 412,
+      innerHeight: 823,
+      outerWidth: 412,
+      outerHeight: 823,
       // In DevTools this value will be exactly 3.
-      devicePixelRatio: '2.625 +/- 1',
+      devicePixelRatio: '1.75 +/- 1',
```
Comment (adamraine) [inline cli/test/smokehouse/test-definitions/fps-max.js:26]: This value should be exact now, no differences?

File: core/test/fixtures/fraggle-rock/artifacts/step0/artifacts.json [hunk 5]
```diff
@@ -163,12 +163,12 @@
     "throttlingMethod": "simulate",
     "screenEmulation": {
       "mobile": true,
-      "width": 360,
```
Comment (adamraine) [inline core/test/fixtures/fraggle-rock/artifacts/step0/artifacts.json:166]: This is probably good for this PR, but we should do another complete artifact refresh to capture any changes the screen adjustment had.

File: report/test/renderer/performance-category-renderer-test.js [hunk 6]
```diff
@@ -160,7 +160,7 @@ describe('PerfCategoryRenderer', () => {
     const oppElements = [...categoryDOM.querySelectorAll('.lh-audit--load-opportunity')];
     expect(oppElements.map(e => e.id).sort()).toEqual(oppAudits.map(a => a.id).sort());
     expect(oppElements.length).toBeGreaterThan(0);
-    expect(oppElements.length).toMatchInlineSnapshot('7');
+    expect(oppElements.length).toMatchInlineSnapshot('6');
```
Comment (adamraine) [inline report/test/renderer/performance-category-renderer-test.js:163]: What was the change here?

File: cli/test/smokehouse/test-definitions/dobetterweb.js (hunk 3 above)
Comment (connorjclark) [reply cli/test/smokehouse/test-definitions/dobetterweb.js:360]: it's unrelated, just so happened that the other audit being tested here got caught up. Don't wanna add it here...cuz it is named specific to its audit... :/

File: report/test/renderer/performance-category-renderer-test.js (hunk 6 above)
Comment (connorjclark) [reply report/test/renderer/performance-category-renderer-test.js:163]: I'm sure it was just image aspect ratio or responsive images no longer failing according to the artifacts.

**Review by adamraine — APPROVED** [review]

File: cli/test/smokehouse/test-definitions/fps-scaled.js [hunk 7]
```diff
@@ -18,12 +18,12 @@ const config = {
...
-      innerHeight: 1280,
-      outerWidth: 360,
-      outerHeight: 640,
+      innerWidth: 824,
+      innerHeight: 1646,
+      outerWidth: 412,
+      outerHeight: 823,
       // In DevTools this value will be exactly 3.
-      devicePixelRatio: '2.625 +/- 1',
+      devicePixelRatio: '1.75 +/- .5',
```
Comment (adamraine) [inline cli/test/smokehouse/test-definitions/fps-scaled.js:26]: Use exact 1.75

**Review by adamraine — CHANGES_REQUESTED** [review]

**Review by adamraine — APPROVED** [review]

**Pull request merged**